frontend/src/components/DropdownListItems.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {Button, Dropdown, Menu} from "antd";
+import {CloseOutlined, LineOutlined, MenuOutlined} from "@ant-design/icons";
+
+const DropdownListItems = ({ListItems}) => {
```

Comment:
Naming convention for variables is camelCase. That would imply that ListItems is a component, but maybe I am confused.

Reply:
you are right, i am just used to have CamelCase variables.